### PR TITLE
Ks/exercises api routes

### PR DESCRIPTION
**Pull request by E-K8**
please review

### CODE REVIEW

File: pages/api/exercises.ts [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+import type { NextApiRequest, NextApiResponse } from "next";
+import query from "../../db";
+import NextCors from "nextjs-cors";
+
+export default async function handler(
+  req: NextApiRequest,
+  res: NextApiResponse
+) {
+  await NextCors(req, res, {
+    // Options
```
Comment (EzyhB) [inline pages/api/exercises.ts]: are the comments still needed?

File: pages/api/exercises.ts [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    methods: ["GET", "HEAD", "PUT", "PATCH", "POST", "DELETE"],
+    origin: "*",
```
Comment (EzyhB) [inline pages/api/exercises.ts:19]: when the front end is deployed, do we need to provide the front end link as the origin for extra security?

File: pages/api/exercises.ts [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    const el = req.body;
+    const data = await query(
+      "INSERT INTO exercises (exercise_name, sets, reps, rest_period, workout_ref) VALUES($1, $2, $3, $4, $5);",
+      [el.exercise_name, el.sets, el.reps, el.rest_period, el.workout_ref]
+    );
+    res.json(data.rows);
+  }
+
+  if (req.method === "DELETE") {
+    const data = await query("DELETE FROM exercises WHERE id = $1;", [
```
Comment (EzyhB) [inline pages/api/exercises.ts]: does this need to be assigned since a delete req doesn't return any data?

File: pages/api/exercises.ts [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+      req.body.id,
+    ]);
+    res.json({ message: "Exercise deleted" });
```
Comment (EzyhB) [inline pages/api/exercises.ts]: maybe specify which exercise was deleted based on the id?

**Review by EzyhB — CHANGES_REQUESTED** [review]

File: pages/api/exercises.ts (hunk 1 above)
Comment (Sagar-Bhil) [reply pages/api/exercises.ts]: No we do not need them anymore. thank you.

File: pages/api/exercises.ts (hunk 2 above)
Comment (Sagar-Bhil) [reply pages/api/exercises.ts:19]: Frontend has not been deployed yet.

File: pages/api/exercises.ts (hunk 3 above)
Comment (Sagar-Bhil) [reply pages/api/exercises.ts]: resolved.

File: pages/api/exercises.ts (hunk 4 above)
Comment (Sagar-Bhil) [reply pages/api/exercises.ts]: deleted exercise is specify with id

File: pages/api/exercises.ts [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+  });
+
+  if (req.method === "GET") {
+    const data = await query("SELECT * FROM exercises");
+    res.json(data.rows);
+  }
+
+  if (req.method === "POST") {
+    const el = req.body;
+    const data = await query(
```
Comment (EzyhB) [inline pages/api/exercises.ts]: youll be receiving an array of objects for exercises, maybe use a for each function to iterate through each exercise? 
- refer back to the populate table function

**Review by EzyhB — CHANGES_REQUESTED** [review]

**Review by EzyhB — APPROVED** [review]

**Pull request merged**